server/src/controllers/internship.ts

```diff
@@ -404,10 +404,38 @@ export async function getAllPaymentTypes(
   } catch (e) {
     return next(e);
   }
-
+  const internships = await Internship.find().lean().select("paymentTypes");
+  const paymentTypes: string[] = [
+    ...new Set(internships.flatMap((internship) => internship.paymentTypes || "")),
+  ];
+  res.json(paymentTypes);
```

Comment:
Das versteh ich nicht ganz. Warum durchsuchst Du hier alle Internships nach ihren Payment Types und callst dann zwei mal res.json?

Reply:
Oh stimmt, an der Stelle ist das Quatsch – ich war grad in dem File und hab gesehen, dass zb bei Intership status gefiltert wird nach Status, die es auch tatsächlich gibt, und hab gedacht, macht bei den anderen Sachen ja auch Sinn, aber wir benutzen das ja noch an anderer Stelle... mach ich rückgängig!